fbfmaproom/fbflayout.py

```diff
@@ -403,32 +403,33 @@ def table_layout():
             dcc.Loading(
                 [
                     table.DataTable(
-                        id="summary",
+                        id="table",
                         page_action="none",
                         style_table={
-                            "height": "auto",
-                            "overflowY": "scroll",
+                            "height": "900px",
                         },
                         css=[
                             {
+                                # attempting to be more specific introduces a bizarre visual artifact
+                                # "selector": "div.dash-fixed-row tr:first-child",
                                 "selector": "tr:first-child",
                                 "rule": "display: none",
                             },
                             {
-                                "selector": "tr:not(last-child)",
-                                "rule": "font-weight: bold; background-color: rgb(255, 255, 255); color: rgb(251, 101, 57);",
-                            },
-                            {
-                                "selector": "tr:last-child",
-                                "rule": "font-weight: bold; background-color: rgb(241, 241, 241); color: black",
+                                "selector": "div.dash-fixed-row tr:nth-child(7)",
+                                "rule": "border-bottom: 2px solid rgb(150, 150, 150)",
                             },
                         ],
                         style_cell={
                             "whiteSpace": "normal",
                             "height": "auto",
                             "textAlign": "center",
                             "border": "1px solid rgb(150, 150, 150)",
-                            "background-color": "rgba(255, 255, 255, 0)",
+                            "backgroundColor": "rgb(248, 248, 248)",
```

Comment:
mistake, remove

Reply:
This is actually correct. I reworked the CSS logic slightly. This is a product of the summary and the data tables being combined, where the gray is the default and the header rows have different background colors